src/processor/k8sattributesprocessor/config.go

```diff
@@ -49,6 +52,24 @@ type Config struct {
 
 	// Section allows to define rules for extracting annotations and labels from Deployment
 	Deployment DeploymentConfig `mapstructure:"deployment"`
+
+	// Section allows to define rules for extracting annotations and labels from StatefulSet
+	StatefulSet StatefulSetConfig `mapstructure:"statefulset"`
+
+	// Section allows to define rules for extracting annotations and labels from ReplicaSet
+	ReplicaSet ReplicaSetConfig `mapstructure:"replicaset"`
+
+	// Section allows to define rules for extracting annotations and labels from DaemonSet
+	DaemonSet DaemonSetConfig `mapstructure:"daemonset"`
+
+	// Section allows to define rules for extracting annotations and labels from StatefulSet
```

Comment:
The comments here and below are probably copy-paste.

Reply:
Yes that is, I fixed the descriptions